paper/joss/reconst_ivim/paper.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Ivim: a model for diffusion and perfusion analysis of MRI'
+tags:
+  - dipy
+  - ivim
+  - diffusion
+  - perfusion
+  - intravoxel incoherent motion
+  - MRI
+  - bi-exponential fitting
+authors:
+ - name: Shahnawaz Ahmed
+   orcid: 0000-0003-1145-7279
+   affiliation: 1
+ - name: Ariel Rokem
+   orcid: 0000-0003-0679-1985
+   affiliation: 2
+ - name: Eric Peterson
+   orcid: 0000-0002-3530-9652
+   affiliation: 3
+ - name: Rafael Neto Henriques
+   orcid: 0000-0000-0000-1234
+   affiliation: 4
+affiliations:
+ - name: Birla Institute of Technology and Science Pilani
+   index: 1
+ - name: The University of Washington eScience Institute
+   index: 2
+ - name: SRI International
+   index: 3
+ - name: MRC Cognition and Brain Sciences Unit, Cambridge, UK
+   index: 4
+date: 6 October 2016
+bibliography: paper.bib
+---
+
+# Summary
+
+Intra-voxel incoherent motion (IVIM) is a technique that aims to separate effects of perfusion (microcirculation of blood in the capillary network) from the effects of molecular diffusion [@le1986mr, @le1988ra] in the signal acquired with a multi-diffusion value MRI acquisition [@le1986mr]. This method was originally developed to study the perfusion and diffusion in the human brain [@le1986mr], but more recently, interest in this technique has expanded and applications have emerged throughout the body [@LeBihan2008] including kidneys, liver, and heart [@Powers1991, @PMID:9038058, @Callot2003, ]. Many more applications are now under investigation such as cancer imaging (prostate, liver, kidney, pancreas, etc.) [@Koh2011] and imaging of the human placenta [@Moore2000, @moore2000vivo]. One of its most common applications still remains brain mapping for neuroscience research
```

Comment:
First sentence:
Intra-voxel incoherent motion (IVIM) is a technique that aims to separate effects of perfusion (microcirculation of blood in the capillary network) from the effects of molecular diffusion in MRI images acquired with a multi-diffusion weights  [@le1986mr, @le1988ra].